### PR TITLE
Feature/rooms

### PR DESCRIPTION
**Pull request by raqso**
Test private room: http://localhost:3000/yP7dQudeJPTijSQ1Bvzg

### CODE REVIEW

File: src/components/ChatRoom/ChatRoom.tsx [hunk 1]
```diff
@@ -14,12 +19,13 @@ export const ChatRoom = () => {
 		dummyRef?.current?.scrollIntoView({ behavior: 'smooth' });
 	});
 
-	const [messages] = useCollectionData<Message & { id: string }>(
-		messagesQuery,
-		{
-			idField: 'id',
-		}
-	);
+	const [, roomId] = window.location.pathname.split('/');
```
Comment (zdzislawbela) [inline src/components/ChatRoom/ChatRoom.tsx:22]: comma?

File: src/models/Room.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	public: boolean;
+}
+
+const create = (data: Partial<Room>) =>
+	firestore.collection('rooms').add({
+		...data,
+		createdAt: TIMESTAMP,
+	});
+
+const isExists = async (roomId: string): Promise<boolean> => {
```
Comment (zdzislawbela) [inline src/models/Room.ts:16]: please explain this name: isExists

File: src/components/ChatRoom/ChatRoom.tsx (hunk 1 above)
Comment (raqso) [reply src/components/ChatRoom/ChatRoom.tsx:22]: Yes, I need a second table element, so I'm destructuring it and assign the second variable only. If I need the 1st one, I'd write something like this:
`const [firstPartOfString, SecondPartOfString] = window.location.pathname.split('/');`

File: src/models/Room.ts (hunk 2 above)
Comment (raqso) [reply src/models/Room.ts:16]: Room's isExists method checks if room exists, pretty straightforward.

**Review by zdzislawbela — APPROVED** [review]

**Pull request merged**